arlo_server/models.py

```diff
@@ -326,3 +325,10 @@ class RoundContestResult(db.Model):
         nullable=False,
     )
     result = db.Column(db.Integer)
+
+
+class File(db.Model):
+    id = db.Column(db.String(200), primary_key=True)
+    name = db.Column(db.String(250), nullable=False)
+    contents = db.Column(db.Text, nullable=False)
+    uploaded_at = db.Column(db.DateTime(timezone=False), nullable=False)
```

Comment:
We'll probably want things like processing_started_at and processing_succeeded_at and possibly processing_errors, but that can wait till first async implementation.

Reply:
My thoughts exactly.